airgun/widgets.py

```diff
@@ -1206,7 +1208,7 @@ def fill(self, value):
     def read(self):
         """Returns string with current widget value"""
         return self.browser.execute_script(
-            "ace.edit('{0}').getValue();".format(self.ace_edit_id))
+            "return ace.edit('{0}').getValue();".format(self.ace_edit_id))
```

Comment:
does this change fix https://github.com/SatelliteQE/airgun/issues/173 ?

Reply:
Yes it fix that issue